SPECS/perl/perl.spec

```diff
@@ -36,19 +40,101 @@
+# cpan/Encode/bin/encguess:             Artistic 2.0
+# cpan/libnet/lib/Net/libnetFAQ.pod:    Artistic    (CPAN RT#117888)
+# cpan/Unicode-Collate/Collate/allkeys.txt:     Unicode
+# inline.h:                             MIT
+# lib/unicore:                          UCD
+# ext/SDBM_File/sdbm.{c,h}:             Public domain
+# regexec.c, regcomp.c:                 HSRL
+# cpan/Locale-Maketext-Simple/lib/Locale/Maketext/Simple.pm:    MIT (with
+#                                       exception for Perl)
+# time64.c:                             MIT
+# perly.h:                              GPLv3+ with Bison exception
+# pod/perlpodstyle.pod:                 MIT
+# pod/perlunicook.pod:                  (GPL+ or Artistic) and Public Domain
+# pod/perlgpl.pod:                      GPL text
+# pod/perlartistic.pod:                 Artistic text
+# ext/File-Glob/bsd_glob.{c,h}:         BSD
+# Other files:                          GPL+ or Artistic
+## Not in a binary package
+# ebcdic_tables.h:                                  MIT
+# cpan/podlators/t/docs/pod.t:                      MIT
+# cpan/podlators/t/docs/pod-spelling.t:             MIT
+# cpan/podlators/t/docs/spdx-license.t:             MIT
+# cpan/podlators/t/docs/synopsis.t:                 MIT
+# cpan/podlators/t/docs/urls.t :                    MIT
+# cpan/podlators/t/lib/Test/RRA.pm:                 MIT
+# cpan/podlators/t/lib/Test/RRA/Config.pm:          MIT
+# cpan/podlators/t/lib/Test/RRA/ModuleVersion.pm:   MIT
+# cpan/podlators/t/style/minimum-version.t:         MIT
+# cpan/podlators/t/style/module-version.t:          MIT
+# cpan/podlators/t/style/strict.t:                  MIT
+# cpan/Term-ANSIColor/t/lib/Test/RRA/Config.pm:     MIT
+## Unbundled
+# cpan/Compress-Raw-Bzip2/bzip2-src:    BSD
+# cpan/Compress-Raw-Zlib/zlib-src:      zlib
+# perl.h (EBDIC parts)                              MIT
+## perl sub-package notice
+# perluniprops.pod is generated from lib/unicore sources:   UCD
+# uni_keywords.h is generated from lib/unicore sources:     UCD
+#
+# This sub-subpackage doesn't contain any copyrightable material.
+# Nevertheless, it needs a License tag, so we'll use the generic
+# "perl" license.
+License:        GPL+ or Artistic
 Epoch:          %{perl_epoch}
-Version:        %{perl_version} 
+Version:        %{perl_version}
 # release number must be even higher, because dual-lived modules will be broken otherwise
-Release:        465%{?dist}
+Release:        487%{?dist}
 Summary:        Practical Extraction and Report Language
 Url:            https://www.perl.org/
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Source0:        https://www.cpan.org/src/5.0/perl-%{perl_version}.tar.xz
 Source3:        macros.perl
-
+#Systemtap tapset and example that make use of systemtap-sdt-devel
+# build requirement. Written by lberk; Not yet upstream.
+#Source4:        perl.stp
+#Source5:        perl-example.stp
 # Tom Christiansen confirms Pod::Html uses the same license as perl
```

Comment:
If we're not using these, I'd remove them:
```suggestion
# Tom Christiansen confirms Pod::Html uses the same license as perl
```

Reply:
Sure, will remove